code/modules/mob/living/carbon/superheroes.dm

```diff
@@ -28,9 +27,10 @@
 		W.flags |= NODROP
 
 /datum/superheroes/proc/assign_genes(var/mob/living/carbon/human/H)
-	if(default_genes.len)
```

Comment:
Global mutable var cannot be declared in type definition, and you are supposed to pass in the proc instead of traits, so the original version never worked. Putting it in proc works..

Also this means superheroes will get genetic damage from instability. However balancing that is a feature - I'll leave that for someone else to do later on.